bezier-rs/lib/src/lib.rs

```diff
@@ -721,18 +726,64 @@ impl Bezier {
 		});
 		result
 	}
+
+	/// Scale will translate a bezier curve a fixed distance away from its original position, and stretch/compress the transformed curve to match the translation ratio.
+	/// Scale takes the following parameters:
+	/// - `distance` - The distance away from the curve that the new one will be scaled to. Positive values will scale the curve in the same direction as the endpoint normals,
+	/// while negative values will scale in the opposite direction.
+	pub fn scale(&self, distance: f64) -> Bezier {
+		assert!(self.is_scalable());
```

Comment:
Maybe put a comment explaining why we need a simple curve. You could even put it in the function description.